### PR TITLE
Configure Crashlytics

### PR DESCRIPTION
**Pull request by cassidyxu**
## Overview

Configured the Firebase Crashlytics script to run.

### Analytics

- Modified the Xcode build configurations
    - Set up Xcode to automatically upload dSYM files

## Test Coverage

- Forced a test crash in order to send a crash report viewable in the Eatery Crashlytics Dashboard.

Screenshot 2024-03-25 at 5 01 38 PM

### CODE REVIEW

File: Eatery Blue/Assets.xcassets/AppIcon.appiconset/appicon_stpatty.png
Comment (MrPeterss) [inline Eatery Blue/Assets.xcassets/AppIcon.appiconset/appicon_stpatty.png:1]: why are we switching back to the st patty's logo?

**Review by MrPeterss — COMMENTED** [review]
Are we changing back to st patty's logo?

File: Eatery Blue.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -995,6 +996,32 @@
...
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+				"${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}",
+				"${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Resources/DWARF/${PRODUCT_NAME}",
+				"${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Info.plist",
+				"$(TARGET_BUILD_DIR)/$(UNLOCALIZED_RESOURCES_FOLDER_PATH)/GoogleService-Info.plist",
+				"$(TARGET_BUILD_DIR)/$(EXECUTABLE_PATH)",
+			);
```
Comment (belle-hu) [inline Eatery Blue.xcodeproj/project.pbxproj:1013]: does this need to be alphabetically ordered? I got a lot of style comments on my PR, but not sure if that applies to crashlytics script tbh.

**Review by vinnie4k — APPROVED** [review]
Great job Cassidy!

**Review by caitlynjin — APPROVED** [review]
Looks good to me!

**Pull request merged**